app/views/shopify_app/partials/_button_styles.html.erb

```diff
@@ -15,12 +14,14 @@
     min-width:3.6rem;
     margin:0;
     padding:0.7rem 1.6rem;
-    background:linear-gradient(to bottom, white, #f9fafb);
-    border:1px solid #c4cdd5;
-    box-shadow:0 1px 0 0 rgba(22, 29, 37, 0.05);
-    border-radius:3px;
+    background-color: #ffffff;
```

Comment:
Are there variables we can use for colors? Or does this all need to be hardcoded?

Reply:
To answer the global comment above, everything has been written myself but copied exactly as is from the Polaris computed styles.
It all needs to be hardcoded as CSS `var()` is [not compatible](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) at all with IE